values/cloudnative-pg/cloudnative-pg-raw.gotmpl

```diff
@@ -0,0 +1,11 @@
+resources:
+  - apiVersion: postgresql.cnpg.io/v1
+    kind: ClusterImageCatalog
+    metadata:
+      name: postgresql
```

Comment:
Now thinking of it, I would use a different name here, e.g.
```suggestion
      name: platform
```

Reply:
applied suggestion